src/metabase/api/activity.clj

```diff
@@ -4,13 +4,9 @@
    [clojure.string :as str]
    [compojure.core :refer [GET]]
    [medley.core :as m]
-   #_{:clj-kondo/ignore [:deprecated-var]}
-   [metabase.api.common
-    :refer [*current-user-id* defendpoint-schema define-routes]]
```

Comment:
I removed the `:refer` so we don't need to put the `ignore` comment in the `ns` declaration, which makes it hard to clean automatically.